src/editor/UI/TreeView.re

```diff
@@ -65,7 +66,8 @@ let itemRenderer =
       backgroundColor(bgColor),
     ];
 
-  let indentStr = String.make(indent * 2, ' ');
+  let explorerIndent = Core.Configuration.getValue(c => c.explorerIndentSize, state.configuration)
```

Comment:
This strategy looks great to me! Thanks for diving in and figuring out how to wire it through to the renderer 👍